From c87710645bff6ea8a20dddc21550fa42a2d148c0 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 13 Jun 2017 09:46:47 +0100
Subject: [PATCH 142/205] crypto: omap-aes - fix spelling mistake "Encryptio"
 -> "Encryption"

Trivial fix to spelling mistake in pr_err message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit 783a5665ca06a71defc0685d9f86be244b5efb1b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/crypto/omap-aes-gcm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/omap-aes-gcm.c b/drivers/crypto/omap-aes-gcm.c
index 521a310..7d4f8a4 100644
--- a/drivers/crypto/omap-aes-gcm.c
+++ b/drivers/crypto/omap-aes-gcm.c
@@ -214,7 +214,7 @@ static int do_encrypt_iv(struct aead_request *req, u32 *tag, u32 *iv)
 		}
 		/* fall through */
 	default:
-		pr_err("Encryptio of IV failed for GCM mode");
+		pr_err("Encryption of IV failed for GCM mode");
 		break;
 	}
 
-- 
1.7.9.5

